### ISSUE TITLE
Usage in Xmonad 

### ISSUE THREAD

**Issue by sugry** [sagb/alttab#129]
This is a report and some wishes (if possible) 
I use Xmonad with options:
```
main :: IO ()
main = do
    dbus <- D.connectSession
    -- Request access to the DBus name
    D.requestName dbus (D.busName_ "org.xmonad.Log")
        [D.nameAllowReplacement, D.nameReplaceExisting, D.nameDoNotQueue]

    xmonad 
        $ docks
        $ dynamicProjects projects
        $ withUrgencyHook NoUrgencyHook 
        $ ewmh 
        $ addDescrKeys' ((mod4Mask, xK_F1), showKeybindings) myKeys
        $ myDesktopConfig { logHook = dynamicLogWithPP $ namedScratchpadFilterOutWorkspacePP (myLogHook dbus) }
    where
      myDesktopConfig = def
        { modMask            = myModMask
        , workspaces         = myWorkspaces
        , borderWidth        = myBorderWidth
        , normalBorderColor  = myNormalBorderColor
        , focusedBorderColor = myFocusedBorderColor
        , focusFollowsMouse  = False
        , clickJustFocuses   = True
        , mouseBindings      = myMouseBindings
        , startupHook        = myStartupHook
        , layoutHook         = myLayoutHook
        , manageHook         = manageHook def <+> myManageHook 
        }
``` 
I launch with `startupHook` with the command 
```
alttab -w 1 -d 2 -vertical -t 900x32 -bg "#263238" -frame "#00BCD4" -bw 2 -bc "#3168A0" -font "xft:MesloLGS NF-11" -i 32x32
```
Alttab finds all icons from the current GTK theme, except for the `urxvt` icon. To help him, posted a picture in `/usr/local/share/icons/hicolor/32x32/apps/urxvt.png`

Everything works great. I really liked the program. But there are some wishes. I would be happy if the developer adds them to TODO .
1. White borders around menu items are not customizable. I would like to replace them with horizontal dividing lines and change the color.
2. I can't change the thickness of the frame border.

p.s. I would also like to be able to remove the border of the frame and instead change the background (highlight) of the active element .

Thanks! 

![Imgur](https://i.imgur.com/yO9wVKL.png)

**Comment by sagb** [comment]
Single-pixel borders are drawn with `-fg` color (same as letters). PR #128 would allow a distinct color, but it needs some fixes before merge (soon).

Thickness of `-frame` border and thickness of single-pixel border are hardcoded and should probably also be customizable (not soon).

"Horizontal dividing lines", highlight by background or other completely new elements are overkill for a small project, sorry.

**Comment by sagb** [comment]
Single-pixel border (around inactive frames was fixed in #132.

**Comment by sugry** [comment]
Thanks, looks great!
